css/properties.json

```diff
@@ -4220,6 +4220,50 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/contain"
   },
+  "contain-intrinsic-size": {
+    "syntax": "[ none | <length> | auto <length> ]{1,2}",
+    "media": "visual",
+    "inherited": false,
+    "animationType": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "percentages": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "groups": [
+      "CSS Containment"
+    ],
+    "initial": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "appliesto": "allElements",
+    "computed": [
+      "contain-intrinsic-width",
+      "contain-intrinsic-height"
+    ],
+    "order": "perGrammar",
+    "status": "experimental",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/contain-intrinsic-size"
+  },
+  "contain-intrinsic-width": {
+    "syntax": "none | <length> | auto <length>",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "byComputedValueType",
+    "percentages": "no",
+    "groups": [
+      "CSS Containment"
+    ],
+    "initial": "none",
+    "appliesto": "elementsForWhichSizeContainmentCanApply",
```

Comment:
This is an enum which has to map to one of the values in https://github.com/mdn/data/blob/main/l10n/css.json and https://github.com/mdn/data/blob/fea9f55167435f34fb39912151e1cc4288e7afb7/css/properties.schema.json#L162. So if this is new (which it looks like it is) then you need to update those files too.

(yes, it's awful. I think this is where you get when you try to represent prose as data. See also https://github.com/mdn/data/issues/585 for some interesting discussion of this.)

Reply:
That is horrible. But then so is this whole thing. I added this value as part of https://github.com/mdn/data/pull/606